dev-tools/audius-compose

```diff
@@ -157,13 +163,15 @@ def build(
 @click.option("-d", "--discovery-provider-replicas", default=3, type=int)
 @click.option("-e", "--elasticsearch-replicas", default=0, type=int)
 @click.option("-b", "--block-explorer", is_flag=True)
+@click.option("-a", "--anti-abuse-oracle", is_flag=True)
```

Comment:
`a` is a very overloaded char, wdyt of making this `-aao`?

Reply:
This is good idea, will think, thanks

Reply:
Having -a would have given us the five options of -a -b -c -d -e

renamed this to -o for now